.github/workflows/reusable-build-test-release.yml

```diff
@@ -1493,7 +1499,12 @@ jobs:
         splunk: ${{ fromJson(needs.meta.outputs.matrix_combinedSplunkversion) }}
         browser: [ "chrome" ]
         vendor-version: ${{ fromJson(needs.meta.outputs.matrix_supportedUIVendors) }}
-
+        python39: [false]
+        marker: ${{ fromJson(inputs.ui_marker) }}
+        include:
+            - splunk: ${{ fromJson(needs.meta.outputs.python39_splunk) }}
+              browser: "chrome"
+              python39: true
```

Comment:
Why this change is included in this PR?

Reply:
strange apparently at rebase I inadvertently added something, it is enough if this line remains:
`marker: ${{ fromJson(inputs.ui_marker) }}` yes?

Reply:
yes, that should be fine